static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -69,7 +69,7 @@
         }
     }
 
-    .advert--large--inverse {
+    .advert--inverse {
```

Comment:
Well done but you have `.advert--large--thumbnail` below on the line 85 and it is the same thing. I should mentioned it properly in the previous comment.